### PR TITLE
use correct params for keystonev3 in dns_fill.py

### PR DESCRIPTION
**Pull request by furlongm**

### CODE REVIEW

File: cookbooks/bcpc/files/default/dns_fill.py [hunk 1]
```diff
@@ -34,14 +34,15 @@ def __init__(self, config):
...
-        auth_params['auth_url'] = config['OS_AUTH_URL'] + \
-            "/%s/" % self.api_version
+        auth_params['auth_url'] = config['OS_AUTH_URL']
         if self.api_version == 'v2.0':
             from keystoneclient.v2_0 import client as kclient
             auth_params['tenant_name'] = config['OS_PROJECT_NAME']
         else:
             from keystoneclient.v3 import client as kclient
             auth_params['project_name'] = config['OS_PROJECT_NAME']
+            auth_params['project_domain_name'] = config['OS_PROJECT_DOMAIN_NAME']
```
Comment (kelvk) [inline cookbooks/bcpc/files/default/dns_fill.py]: root@bcpc-dev-r1n1:~# /usr/local/bin/dns_fill.py -c /usr/local/etc/dns_fill.yml dump
Traceback (most recent call last):
  File "/usr/local/bin/dns_fill.py", line 271, in <module>
    args.func(args)
  File "/usr/local/bin/dns_fill.py", line 244, in c_dump
    dnsp = dns_popper(config)
  File "/usr/local/bin/dns_fill.py", line 44, in __init__
    auth_params['project_domain_name'] = config['OS_PROJECT_DOMAIN_NAME']
KeyError: 'OS_PROJECT_DOMAIN_NAME'

File: cookbooks/bcpc/files/default/dns_fill.py [hunk 2]
```diff
@@ -34,14 +34,15 @@ def __init__(self, config):
             'password': config['OS_PASSWORD'],
             'insecure': True
         }
-        auth_params['auth_url'] = config['OS_AUTH_URL'] + \
-            "/%s/" % self.api_version
+        auth_params['auth_url'] = config['OS_AUTH_URL']
```
Comment (kelvk) [inline cookbooks/bcpc/files/default/dns_fill.py]: API version needs to be indicated in URL, either in here or in configuration.

Comment (furlongm) [reply cookbooks/bcpc/files/default/dns_fill.py]: This commit depends on feature/l-to-m-migration branch, specifically this commit, which changes the OS_AUTH_URL format and adds the other variables: ee5bcae4ff584bab6574a106b7fd03f009bdf7a0

**Comment by furlongm** [comment]
Pushed to l-to-m-migration

**Pull request closed without merging**